### PR TITLE
updated to Rails 4.2.0

### PR DESCRIPTION
**Pull request by AlexMcConnell**

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 ActiveRecord::Schema.define(version: 20141005040238) do
 
-  create_table "widgets", force: true do |t|
+  create_table "widgets", force: :cascade do |t|
```
Comment (sb8244) [inline db/schema.rb:16]: huh, I haven't seen this before. Do you know what this does or was it automatic?

File: .gitignore [hunk 2]
```diff
@@ -6,6 +6,7 @@
 
 # Ignore bundler config.
 /.bundle
+/.ruby-version
```
Comment (sb8244) [inline .gitignore]: Let's leave .ruby-version because it lowers friction

File: db/schema.rb (hunk 1 above)
Comment (AlexMcConnell) [inline db/schema.rb:16]: It was automatic, but I did notice it and looked it up. It's actually the first thing listed on the notable changes for 4.2.0.
http://guides.rubyonrails.org/4_2_release_notes.html#active-record-notable-changes

Comment (sb8244) [inline db/schema.rb:16]: Nice! Thanks for the info

**Comment by sb8244** [comment]
Thanks @wtiger ! Appreciate it :)

**Pull request merged**